resource-sets/schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Resource Set",
+  "type": "object",
+  "properties": {
+    "resources": {
```

Comment:
This should definitely have the property "version" as an optional element.

This should maybe have name and description? I could see a universe where people want to like keep track of different sets.

Reply:
Actually I have switched from "maybe have name and description" to "probably have name and description"

Reply:
Added.

Reply:
We have made version mandatory on every recipe so I'm wondering if it's confusing to also have version at the top level. But I guess that would control whether for examples `images` is allowed as a top-level field?

Reply:
It's confusing, but I think there's potential value in it.